playground/components/Playground.js

```diff
@@ -114,7 +121,7 @@ class Playground extends React.Component {
           {
             dataStructures.length > 0 &&
               <JsonFormatterComponent
-                element={this.state.parseResult.dataStructures[0]}
+                element={dataStructures.get(0)}
```

Comment:
We can use `.first` here, which is more semantic. Another thing: we only getting the first category here if I see it right. Cannot we just process all the `dataStructures` categories ?

Reply:
We could but as we don't have any real use-case yet (the used apib parser generate only a single category I'd vote to leave the code as close to what it was before introducing minim to the mix. We can extend later as soon as we need it.